test/torchaudio_unittest/functional/torchscript_consistency_impl.py

```diff
@@ -548,6 +548,22 @@ def func(tensor):
         tensor = common_utils.get_whitenoise(sample_rate=44100)
         self._assert_consistency(func, tensor)
 
+    def test_apply_codec(self):
+
+        def func(tensor):
+            sample_rate = 8000,
+            format = "wav",
+            compression = None,
```

Comment:
The trailing commas here make the TorchScript compiler think that they are `tuple` type, thus the test is failing. Removing the commas should make the test work.